backend/dataall/base/utils/enum_utils.py

```diff
@@ -0,0 +1,26 @@
+from dataall.base.api import gql
+
+
+def generate_enum_resolver(enum_class):
+    def resolver(context, source):
+        return [{'name': item.name, 'value': item.value} for item in enum_class]
+
+    return resolver
+
+
+EnumItem = gql.ObjectType(
+    name='EnumItem',
+    fields=[
+        gql.Field(name='name', type=gql.String),
+        gql.Field(name='value', type=gql.String),
+    ],
+)
+
+
+def generate_enum_query(enum_class, test_scope):
```

Comment:
what's the `test_scope`?

Reply:
QueryField has the parameter "test_scope". It means, the this part is loaded, when we test this module AFAIK